tools/all_questions.xml

```diff
@@ -1924,6 +1924,40 @@ Ebenso wie bei den Formaten gilt hier: je standardisierter, offener und etablier
 		<optionsets/>
 		<conditions/>
 	</question>
+	<question dc:uri="https://rdmorganiser.github.io/terms/questions/ua_ruhr_v1_1_0-beta_sonja/data-usage/data-sharing-and-re-use-publication/repository">
+		<uri_prefix>https://rdmorganiser.github.io/terms</uri_prefix>
+		<key>repository</key>
+		<path>
+		ua_ruhr_v1_1_0-beta_sonja/data-usage/data-sharing-and-re-use-publication/repository
+		</path>
+		<dc:comment/>
+		<attribute dc:uri="https://rdmorganiser.github.io/terms/domain/project/dataset/sharing/repository"/>
```

Comment:
Dont we need a new attribute see  https://github.com/rdmorganiser/rdmo-catalog/issues/55#issuecomment-866776185

Reply:
Add attribute to domain/rdmo.xml  with short comment including link to discussion https://github.com/rdmorganiser/rdmo-catalog/issues/55